LICENSE-MIT

```diff
@@ -0,0 +1,23 @@
+
+Copyright (c) 2015 Davide Di Criscito.
```

Comment:
Uhm...in this case 2016 Zeno Rocha and we don't need the file since MIT license is on Readme.